brax/training/agents/ppo/train.py

```diff
@@ -336,7 +335,7 @@ def f(carry, unused_t):
     # Update normalization params and normalize observations.
     normalizer_params = running_statistics.update(
         training_state.normalizer_params,
-        data.observation if ndarray_obs else data.observation['state'],
+        data.observation,
```

Comment:
Can you add a test in train_test.py that actually makes use of asymmetric obs? It's hard to tell why you chose this impl without seeing it used in practice

Reply:
Please fix

Reply:
Added!